downstream/modules/platform/proc-aap-controller-backup.adoc

```diff
@@ -0,0 +1,42 @@
+[id="aap-controller-backup"]
+
+= Backing up the {ControllerNameStart} deployment
+
+[role=_abstract]
+Use this procedure to back up a deployment of the controller, including jobs, inventories, and credentials.
+
+.Prerequisites
+
+* You must be authenticated with an Openshift cluster.
+* The {OperatorPlatform} has been deployed to the cluster.
+* The {PlatformName} is deployed to using the operator.
```

Comment:
This renders as:
```
The Red Hat Ansible Automation Platform is deployed to using the operator.
```

It should instead read:
```
Automation Controller is deployed to using the AAP Operator.
```

Reply:
Fixed.